src/Services/Air/AirParser.js

```diff
@@ -576,6 +580,103 @@ function gdsQueue(req) {
+
+      const exchangeBundleTotal = json['air:AirExchangeBundleTotal'];
+      const totalBundle = exchangeBundleTotal
+        && exchangeBundleTotal[exchangeInfoRoot];
+
+      const exchangeDetails = json['air:AirExchangeBundle'].map((bundle) => {
+        const exchange = bundle[exchangeInfoRoot];
+        const taxes = exchange[`common_${this.uapi_version}:PaidTax`]
+          .map(tax => ({ type: tax.Code, value: tax.Amount }));
+
+        return {
+          ...format.formatAirExchangeBundle(exchange),
+          taxes,
+          uapi_pricing_info_ref: Object.keys(bundle['air:AirPricingInfoRef'])[0],
+        };
+      });
+
+      const solution = utils.firstInObj(json['air:AirPricingSolution']);
+      const segments = Object.keys(solution['air:AirSegment']).map((key) => {
+        const segment = solution['air:AirSegment'][key];
+        return {
+          ...format.formatSegment(segment),
+          serviceClass: segment.CabinClass,
+          bookingClass: segment.ClassOfService,
+        };
+      });
+
+      const pricingInfo = Object.keys(solution['air:AirPricingInfo'])
+        .map((key) => {
+          const pricing = solution['air:AirPricingInfo'][key];
+
+          const bookingInfo = Object.keys(pricing['air:BookingInfo'])
+            .map((bookingInfoKey) => {
+              const info = pricing['air:BookingInfo'][bookingInfoKey];
+              const fare = pricing['air:FareInfo'][info.FareInfoRef];
+
+              return {
+                bookingCode: info.BookingCode,
+                cabinClass: info.CabinClass,
+                baggage: format.getBaggage(fare['air:BaggageAllowance']),
+                fareBasis: fare.FareBasis,
+                from: fare.Origin,
+                to: fare.Destination,
+                uapi_segment_ref: info.SegmentRef,
+              };
+            });
+
+          return {
+            ...format.formatPrices(pricing),
+            bookingInfo,
+            uapi_pricing_info_ref: pricing.Key,
+            fareCalc: pricing['air:FareCalc'],
+          };
+        });
+
+      const airPricingDetails = solution['air:PricingDetails'];
+
+      return {
+        exchangeToken: zippedToken,
+        exchangeDetails,
+        segments,
+        pricingInfo,
+        pricingDetails: {
+          pricingType: airPricingDetails.PricingType,
+          lowFareFound: airPricingDetails.LowFareFound === 'true',
+          lowFarePricing: airPricingDetails.LowFarePricing === 'true',
+          discountApplies: airPricingDetails.DiscountApplies === 'true',
+          penaltyApplies: airPricingDetails.DiscountApplies === 'true',
+          conversionRate: parseFloat(airPricingDetails.ConversionRate || 1),
+          rateOfExchange: parseFloat(airPricingDetails.RateOfExchange || 1),
+          validatingVendor: airPricingDetails.ValidatingVendorCode,
+        },
+        pricingSolution: { ...format.formatPrices(solution) },
```

Comment:
Why
```javascript
pricingSolution: { ...format.formatPrices(solution) },
```

instead of just

```javascript
pricingSolution: format.formatPrices(solution),